extensions/stackdriver/stackdriver.cc

```diff
@@ -36,14 +36,23 @@ namespace Plugin {
 namespace Stackdriver {
 
 using namespace opencensus::exporters::stats;
+using namespace google::protobuf::util;
+using namespace stackdriver::config;
 
 constexpr char kStackdriverExporter[] = "stackdriver_exporter";
 constexpr char kExporterRegistered[] = "registered";
 
 void StackdriverRootContext::onConfigure(
     std::unique_ptr<WasmData> configuration) {
-  // TODO: Add config for Stackdriver plugin, such as reporter kind.
-  UNREFERENCED_PARAMETER(configuration);
+  // Parse configuration JSON string.
+  JsonParseOptions json_options;
+  Status status =
+      JsonStringToMessage(configuration->toString(), &config_, json_options);
```

Comment:
I think you can use absl::string_view here (`configuration->view()`)

Reply:
Tried but it does not seem to work:
```
extensions/stackdriver/stackdriver.cc:50:7: error: no matching function for call to 'JsonStringToMessage'
      JsonStringToMessage(configuration->view(), &config_, json_options);
      ^~~~~~~~~~~~~~~~~~~
external/com_google_protobuf/src/google/protobuf/util/json_util.h:107:30: note: candidate function not viable: no known conversion from 'StringView' (aka 'absl::string_view') to 'google::protobuf::StringPiece' for 1st argument
PROTOBUF_EXPORT util::Status JsonStringToMessage(
                             ^
external/com_google_protobuf/src/google/protobuf/util/json_util.h:110:21: note: candidate function not viable: requires 2 arguments, but 3 were provided
inline util::Status JsonStringToMessage(StringPiece input,
                    ^
```

Reply:
Ah, ok. Thanks. I think protobuf hasn't yet pushed StringPiece->string_view update.